frontend/src/screens/participant-home.tsx

```diff
@@ -42,7 +42,8 @@ const StudyCard:React.FC<{ study: ParticipantStudy }> = ({ study }) => {
                 <Box className="card-body" direction="column">
                     <Box justify="between">
                         <h5 className="card-title">{study.title}</h5>
-                        {study.completedAt && <Icon icon="checkCircle" css={{ flex: '0 0 25px' }} color="green" />}
+                        {study.completedAt && study.consentGranted && <Icon icon="checkCircle" css={{ flex: '0 0 25px' }} color='green' />}
+                        {study.completedAt && !study.consentGranted && <Icon icon="plusCircle" css={{ flex: '0 0 25px' }} color='orange' />}
                     </Box>
```

Comment:
It'd be a bit cleaner to leave the single line and only change the color:
```jsx
{study.completedAt && <Icon icon="checkCircle" color={study.consentGranted ? 'green' : 'orange'} css={{ flex: '0 0 25px' }}  />}
```

Reply:
Well, I did that at first, but it was pointed out to me that it is not color-blind accessible that way (Thanks Phil!) Figured this part is about to be rewritten. Where do you get valide valuse for Icon, anyway?

Reply:
well they're typescript so it has completion which is what I normally rely on, but in this case it's in the icon component: https://github.com/openstax/kinetic/blob/main/frontend/src/components/icon.tsx#L26-L35  That's only some of them though, note that the interface extends `IconifyIcon` so there's additional ones in there.  Typically I use `jump to definition` to find stuff like that, because it's somewhere in `node_modules`

Reply:
Yeah I wanted to use a yellow "caution triangle"